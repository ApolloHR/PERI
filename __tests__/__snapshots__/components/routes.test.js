import React from 'react';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

import Routes from '../../../Client/src/components/routes.jsx';

describe('>>>R O U T E S --- Snapshot', () => {
  it('+++capturing Snapshot from routes', () => {
    const renderedValue = renderer.create(<Routes/>).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});