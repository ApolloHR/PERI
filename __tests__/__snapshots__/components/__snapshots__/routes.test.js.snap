// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>R O U T E S --- Snapshot +++capturing Snapshot from routes 1`] = `
<div>
  <div
    className="container"
  >
    <nav
      className="level"
    >
      <div
        className="level-left"
      >
        <i
          className="fa fa-ravelry fa-2x has-text-primary"
          style={
            Object {
              "marginBottom": 10,
              "marginTop": 20,
            }
          }
        />
      </div>
      <div
        className="level-right"
      >
        <p
          className="level-item has-text-centered"
        >
          <a
            activeClassName="active"
            className="has-text-grey-dark"
            href="/"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": 10,
                "marginRight": 10,
              }
            }
          >
             Home 
          </a>
        </p>
        <p
          className="level-item has-text-centered"
        >
          <a
            activeClassName="active"
            class="has-text-grey-dark"
            href="/buildTrip"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": 10,
                "marginRight": 10,
              }
            }
          >
             Build Trip 
          </a>
        </p>
        <p
          className="level-item has-text-centered"
        >
          <a
            activeClassName="active"
            class="has-text-grey-dark"
            href="/cart"
            onClick={[Function]}
            style={
              Object {
                "marginLeft": 10,
                "marginRight": 10,
              }
            }
          >
             Cart 
          </a>
        </p>
        <p
          className="level-item has-text-centered"
        >
          <a
            activeClassName="active"
            class="has-text-grey-dark"
            href="/auth/google"
            style={
              Object {
                "marginLeft": 10,
                "marginRight": 10,
              }
            }
          >
             G-Login 
          </a>
        </p>
        <p
          className="level-item has-text-centered"
        >
          <a
            activeClassName="active"
            class="has-text-grey-dark"
            href="/auth/facebook"
            style={
              Object {
                "marginLeft": 10,
                "marginRight": 10,
              }
            }
          >
             FB-Login 
          </a>
        </p>
      </div>
    </nav>
  </div>
</div>
`;
